WordPress/gutenberg
# Add 'Reset' option to MediaReplaceFlow component

## What? and Why?

`Reset` option is used to clear media from block, such as image block, cover block etc. However, the `MediaReplaceFlow` component does not by default have the `Reset` option available. We used to add the `Reset` option as a child of the `MediaReplaceFlow` component, Whenever necessary. We had to include the `Reset` option in the `MediaReplaceFlow` component itself because it is a fundamental feature that is needed anywhere media is used, saving us from having to write additional code.
 
## How?

This PR,
- Adds `Reset` option to `MediaReplaceFlow` Component.
- Update implementation of `MediaReplaceFlow` component in `Image`, `Cover`, `Post Featured Image`, `Site Logo` blocks
- Adds `Reset` option to `Audio`, `Video`, `File` and `Media-Text` block.
- Also, Update implementation of `MediaReplaceFlow` component in background support under global styles.

## Testing Instructions
- Edit post
- Add block from any of ( `Image`, `Cover`, `Post Featured Image`, `Site Logo` , `Audio`, `Video`, `File`, `Media-Text` )
- Select media into added block
- Ensure `Reset` option is visible in toolbar
- Ensure media is removed on click of `Reset`

### Testing Instructions for Keyboard


## Screenshots or screencast 


| Block  | Reset option demo |
| ------------- | ------------- |
| Image  |  ![image-block](https://github.com/user-attachments/assets/b6a47c0c-7bae-4d07-aaf8-65610635a7bd) |
| Cover  | ![cover-block](https://github.com/user-attachments/assets/d345486a-0f33-4da7-b16d-77c4bfa70f3d)  |
| Post Featured Image  | ![featured-image-block](https://github.com/user-attachments/assets/6a8b49bd-9015-4b47-a3f6-8f98a5522f3f)  |
| Site Logo  | ![site-logo](https://github.com/user-attachments/assets/a27458b9-4e57-45ef-8959-f7c45874abe2)  |
| Audio  | ![audio-block](https://github.com/user-attachments/assets/c2d7cdc5-999f-4bd6-99a4-71e5df691195)  |
| Video  | ![video-block](https://github.com/user-attachments/assets/dd893e32-ceae-42f5-b763-d3090fcc2fef)  |
| File  | ![file-block](https://github.com/user-attachments/assets/1f41bf7c-7061-418b-aae0-9d9d66645b44)  |
| Media Text  | ![media-text-block](https://github.com/user-attachments/assets/e0d0062b-87bb-4505-a292-729aa5d9af62) |
